integration-tests/tests/src/typings.d.ts

```diff
@@ -29,7 +29,80 @@ interface path {
 
 type Require = (id: string) => unknown;
 
-type Environment = Record<string, unknown>;
+type Environment = Record<string, unknown> & {
+  longTimeout?: number;
```

Comment:
We could add a comment to it (feel free to modify it).
```suggestion
  /** Set the number of milliseconds to use for the long timeout. */
  longTimeout?: number;
```

Reply:
I added docs for all the other variables, but forgot to add it for the one I was actually adding 🙈

Reply:
😛